utils/include_test.py

```diff
@@ -0,0 +1,46 @@
+#!/usr/bin/env python3
+"""Check that all daos headers can be included stand-alone"""
+
+import json
+import os
+import subprocess  # nosec
+import tempfile
+
+
+def check_dir(include_dir, sub_dir):
+    """Check all files in one directory"""
+
+    h_dir = include_dir
+
+    if sub_dir:
+        h_dir = os.path.join(include_dir, sub_dir)
+    for entry in sorted(os.listdir(h_dir)):
+        if not os.path.isfile(os.path.join(h_dir, entry)):
+            check_dir(include_dir, entry)
```

Comment:
This only works recursively once, right? But that's okay because our includes aren't deep

Reply:
Correct.